devbox.go

```diff
@@ -236,5 +238,29 @@ func missingDevboxJSONError(dir string) error {
 			}
 		}
 	}
-	return usererr.New("No devbox.json found in %s. Did you run `devbox init` yet?", dir)
+	return usererr.New("No devbox.json found in %s, or any parent directories. Did you run `devbox init` yet?", dir)
+}
+
+func findConfigDir(dir string) (string, error) {
+
+	cur := dir
+	if cur == "" || cur == "." {
+		var err error
+		cur, err = os.Getwd()
+		if err != nil {
+			return "", errors.WithStack(err)
+		}
+	}
```

Comment:
I think you need to call `filepath.Abs` to handle all the relative path cases. For example, if the config is in `./repo` and the user runs `devbox ..` from `./repo/a`.

Reply:
The `filepath.Abs` was being called inside `pathArgs`: https://github.com/jetpack-io/devbox/blob/7ae5af019d7f568bb50e164401fa6b293a22e7c5/boxcli/args.go#L18
which is then passed in here.

But I do like the suggestion to sanitize the `dir` here as well via calling `filepath.Abs`

Reply:
Yeah, it might be a good idea to sanitize it here too. Otherwise it's not super obvious that this method requires it.